Makefile

```diff
@@ -2008,7 +2008,7 @@ manila_deploy_prep: manila_deploy_cleanup ## prepares the CR to install the serv
 manila_deploy: input manila_deploy_prep ## installs the service instance using kustomize. Runs prep step in advance. Set CINDER_REPO and CINDER_BRANCH to deploy from a custom repo.
 	$(eval $(call vars,$@,manila))
 	make wait
-	# bash scripts/operator-deploy-resources.sh
+	bash scripts/operator-deploy-resources.sh
```

Comment:
not sure why this was commented, but removing the comment looks the right thing to do here